From 9c943319611e4fb87c2656c5b128ebd9771b7279 Mon Sep 17 00:00:00 2001
Message-Id: <9c943319611e4fb87c2656c5b128ebd9771b7279.1413837856.git.chang-joon.lee@intel.com>
In-Reply-To: <006497b7323eca32c2f489a1a9f662743c0d16d2.1413837855.git.chang-joon.lee@intel.com>
References: <006497b7323eca32c2f489a1a9f662743c0d16d2.1413837855.git.chang-joon.lee@intel.com>
From: Jon Bloomfield <jon.bloomfield@intel.com>
Date: Mon, 19 May 2014 12:41:27 +0100
Subject: [PATCH 017/312] REVERTME [VPG]: drm/i915: Disable command parser by
 default

The command parser functionality is currently incomplete because
the upstream drm now has partial support and this causes conflicts
with the internal VPG patches.

Disabling the parser for now until the remaining VPG patches can
be reworked.

Change-Id: Ic440e8ffe71d2cd23bde6df75848c0358c07dfa8
Signed-off-by: Jon Bloomfield <jon.bloomfield@intel.com>
---
 drivers/gpu/drm/i915/i915_params.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/i915/i915_params.c b/drivers/gpu/drm/i915/i915_params.c
index d05a2af..aba1f83 100644
--- a/drivers/gpu/drm/i915/i915_params.c
+++ b/drivers/gpu/drm/i915/i915_params.c
@@ -46,7 +46,7 @@ struct i915_params i915 __read_mostly = {
 	.reset = true,
 	.invert_brightness = 0,
 	.disable_display = 0,
-	.enable_cmd_parser = 1,
+	.enable_cmd_parser = 0,
 	.disable_vtd_wa = 0,
 };
 
-- 
1.7.9.5

